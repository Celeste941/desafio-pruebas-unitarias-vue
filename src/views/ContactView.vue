<template>
  <section>
    <h1>Secci√≥n de Contacto</h1>
  </section>
</template>

<script>
export default {
  name: 'ContactView'
}
</script>
