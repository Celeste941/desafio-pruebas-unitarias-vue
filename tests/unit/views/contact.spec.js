import { mount } from "@vue/test-utils";
import ContactView from "@/views/ContactView.vue";

describe("Test de vista ContacView.vue", () => {
  it("Match de snapshot con ContacView.vue", () => {
    const wrapper = mount(ContactView);
    expect(wrapper.html()).toMatchSnapshot();
  });
});