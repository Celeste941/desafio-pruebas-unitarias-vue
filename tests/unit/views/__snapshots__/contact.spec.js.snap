// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test de vista ContacView.vue Match de snapshot con ContacView.vue 1`] = `
<section>
  <h1>Secci√≥n de Contacto</h1>
</section>
`;
